coral/src/app/features/requests/schemas/SchemaRequests.tsx

```diff
@@ -57,14 +65,17 @@ function SchemaRequests() {
       currentStatus,
       currentTopic,
       showOnlyMyRequests,
+      currentType,
     ],
     queryFn: () =>
       getSchemaRequests({
         pageNo: String(currentPage),
         env: currentEnvironment,
-        requestStatus: currentStatus,
+        requestStatus:
+          currentStatus !== defaultStatus ? currentStatus : undefined,
```

Comment:
When we do the "consistency review" I think we should also look at the approaches taken to handle the cases where `"ALL"` is not a valid value for a param.

Because for example, in the ACL case, we allow passing `"ALL"` for params for which it is not a valid value, and it is omitted in this way at the `domain` level :o https://github.com/aiven/klaw/blob/main/coral/src/domain/acl/acl-api.ts#L33

And it would be nice to have a unified way of tackling this, so that it's less confusing for new people / for us in a few months :o

Reply:
There's an issue for that already :D 
https://github.com/aiven/klaw/issues/900

(I added that in the description)

Reply:
Ugh, you're right, I thought I added that 🤔  It should be an explicit follow up issue! I'll create and link one.